openmetrics/src/main/java/io/airlift/openmetrics/MetricsConfig.java

```diff
@@ -39,7 +39,7 @@ public MetricsConfig setJmxObjectNames(List<String> jmxObjectNames)
 
         for (String jmxObjectName : jmxObjectNames) {
             try {
-                objectNames.add(new ObjectName(jmxObjectName));
+                objectNames.add(new ObjectName(jmxObjectName.replace("\\", "")));
```

Comment:
I wonder if we could be more generic and support removing the escaping characters in the configuration module.